test/parser/mixins.spec.js

```diff
@@ -25,6 +25,14 @@ describe('Parser', () => {
             expect(root.first.type).to.eql('atrule');
         });
 
+        it('should parse parametric mixins', () => {
+            const root = parse('.foo(@bar) { color: @bar;}', {mixinsAsAtRules: true, innerMixinsAsRules: true});
+
+            expect(root.first.type).to.eql('atrule');
+            expect(root.first.params[0].type).to.eql('atrule');
```

Comment:
@patsissons, what do you think about it, especially from Stylelint side?

I propose to add custom `MixinNode` (similar to AtRule or Rule in PostCSS), `MixinParam` (maybe similar to Declaration in PostCSS), `ExtendNode` (~Rule) nodes and extend stringifier of `postcss-less`.